### PR TITLE
Added non-pollute property

### PR DESCRIPTION
**Pull request by Qix-**
@sindresorhus this should address #10.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -33,3 +33,10 @@ it('should group related codes into categories', function () {
 it('groups should not be enumerable', function () {
 	assert.equal(Object.keys(ansi).indexOf('modifiers'), -1);
 });
+
+it('should not pollute other objects', function () {
+	var obj1 = require('./');
+	var obj2 = require('./');
+	obj1.foo = true;
+	assert.equal(!!obj2.foo, false);
```
Comment (jbnicolai) [inline test.js]: `assert.unequal(obj1.foo, obj2.foo)` seems simpler/more intuitive

Comment (Qix-) [inline test.js]: I'll add it here in a sec.

Comment (Qix-) [inline test.js]: Nvm, I see you got it haha.

Comment (jbnicolai) [inline test.js]: yep :)

**Pull request merged**